src/lib.rs

```diff
@@ -43,18 +55,21 @@ where
             spi,
             end_frame_length: 4,
             invert_end_frame: true,
+            pixel_type: PixelType::BGR,
```

Comment:
Maybe document the default type & that you can use `new_with_custom_postamble` for setting it

Reply:
Updated with documentation. It felt a little weird to put this on the same constructor that controls postamble frame settings. Would you rather these be split somehow?

Reply:
Splitting would necessitate moving to a builder pattern, which i think is not yet necessary. I think renaming the function is enough, maybe something in the direction of `new_with_options`?

Reply:
Sounds good. Updated.